src/test/java/io/airlift/compress/thirdparty/JdkInflateDecompressor.java

```diff
@@ -29,7 +29,7 @@ public int decompress(byte[] input, int inputOffset, int inputLength, byte[] out
             throws MalformedInputException
     {
         try {
-            Inflater inflater = new Inflater();
+            Inflater inflater = new Inflater(true);
```

Comment:
What's broken (add some details in the commit message)? This is not very obvious.